tools/scylla-nodetool.cc

```diff
@@ -3438,6 +3439,7 @@ Fore more information, see: https://opensource.docs.scylladb.com/stable/operatin
                 "sstableinfo",
                 "Information about sstables per keyspace/table",
 R"(
+Fore more information, see: https://opensource.docs.scylladb.com/stable/operating-scylla/nodetool-commands/sstableinfo.html
```

Comment:
s/Fore/For/. 

We need to make the links refer to the version for when they were built.

Let's have a doc_link() function. fmt::format("Fore more information, see {}", doc_link("operating-scylla/nodetool-commands/sstableinfo.html"). The doc_link() function can use information from SCYLLA-VERSION-GEN (via release.cc) to get the rest of the URL and the version reference. For later of course.

/cc @annastuchlik

Reply:
Another alternative is using the same source to generate both docs and the CLI --help output
Here is one way to do so
https://github.com/scylladb/scylla-manager/pull/2987

Reply:
Fixed the typo. I also fixed it for the other commands, copy-pasta spreads fast.

Reply:
As for the improvement, I will follow-up with the suggested `doc_link()` method.